projects/ngx-openlayers/src/lib/sources/xyz.component.ts

```diff
@@ -41,15 +43,15 @@ export class SourceXYZComponent extends SourceComponent implements AfterContentI
   @Input()
   maxZoom: number;
   @Input()
-  tileGrid: tilegrid.TileGrid;
+  tileGrid: any;
```

Comment:
I think that this shoul be `tileGrid: TileGrid;` with   `import { TileGrid } from 'ol/tilegrid';`